src/cmd/protofix/protofix.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import(
+	"os"
+	"fmt"
+
+	"github.com/pachyderm/pachyderm/src/pkg/protofix"
+)
+
+func usage() {
+	fmt.Println("usage: protofix fix <rootdirectory>")
+	fmt.Println("usage: protofix revert <rootdirectory>")
+}
+
+func main() {
+
+	if len(os.Args) < 3 {
+		fmt.Println("Missing root directory")
+		usage()
+		os.Exit(1)
+	}
+
+	if os.Args[1] == "fix" {
+		protofix.FixAllPBGOFilesInDirectory(os.Args[2])
+	}
+	if os.Args[1] == "revert" {
+		protofix.RevertAllPBGOFilesInDirectory(os.Args[2])
+	}
+
+}
```

Comment:
If functionally balloons here we should move this over to using cobra our other binaries. But no need for now.